### PR TITLE
Add podman 1.3.1 cask

### PR DESCRIPTION
**Pull request by ssbarnea**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/podman.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'podman' do
+  version '1.3.1'
+  sha256 '438f7788fe28e88ae800e99896eb42a07775682e4bc69a2f618fc2c36c8a4a21'
+
+  url "https://github.com/containers/libpod/releases/download/v#{version}/podman-v#{version}.dmg"
+  appcast 'https://github.com/containers/libpod/releases.atom'
+  name 'podman'
+  homepage 'https://github.com/containers/libpod/'
+
+  app 'podman'
```
Comment (vitorgalvao) [inline Casks/podman.rb]: This is not an app, it’s a binary.

**Comment by vitorgalvao** [comment]
This seems to be a CLI-only open-source tool. Please submit it first to [Homebrew/core](https://github.com/homebrew/homebrew-core) as a formula. Return here if not accepted (linking back to the pull request where it was refused).

This policy is documented in [faq/rejected_casks](https://github.com/homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> The app is both open-source and CLI-only (i.e. it only uses the `binary` artifact). In that case, and [in the spirit of deduplication](https://github.com/Homebrew/homebrew-cask/issues/15603), submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core) as a formula that builds from source. If it is rejected, you may then try again as a cask (link us to the issue so we can see the discussion and reasoning for rejection).

**Comment by ssbarnea** [comment]
@victorgalvo There is a reason why we ended up creating a cask instead of a formula: building podman under macos is still a real challenge because it needs linux kernel code. For some time we will need to use binary distribution approach but be sure that we will do our best to address the compilation issue (far from trivial).

**Comment by jrop** [comment]
@vitorgalvao ^^^ (tagging you since the previous comment did not)

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**